### PR TITLE
feat(layout): Add phase banner

### PR DESCRIPTION
**Pull request by jpveooys**
This adds a beta phase banner across the entire site. It links to our email address.

I had to place it inside `<header>` to avoid https://dequeuniversity.com/rules/axe/4.1/region. This seems to be the way GOV.UK is going as well: https://github.com/alphagov/govuk-frontend/pull/2824#issuecomment-1387248436.

There was also some complexities with styling as the bottom border of the phase banner should not be shown when the primary nav is visible. A `moduk--primary-nav-expanded` class is used on `<body>` for this (dynamically added and removed on mobile when the menu is opened and closed).

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
   rules: {
     'subject-case': [1, 'always', 'sentence-case'],
     'body-max-line-length': [0],
+    'footer-max-line-length': [0],
```
Comment (jpveooys) [inline commitlint.config.js:11]: This was being flagged on my commit message (probably confused by the colons). Disabled it as we haven't been using footers.

**Comment by adamledwards** [comment]
Nice works well, only one comment. 
On mobile there is a border 
image

**Comment by jpveooys** [comment]
@adamledwards Thanks, that's another case where the border needs to be hidden :disappointed: Have pushed a fixup (open to suggestions for how to improve it...)

**Review by adamledwards — APPROVED** [review]
looks good

**Pull request merged**